CONTRIBUTING.md

```diff
@@ -50,16 +50,17 @@ system already, but many of them are preinstalled on a Pivotal dev box.
 
 (Run all of these commands from the project root)
 
-ruby 2.1.2 
+ruby 2.1.2
 
-bundle
+bundler
 
-  $ gem install bundle
-    
-node and npm 
+  $ gem install bundler
+
+node and npm (at this time `nodejs` version `0.10` is required. Install
+[nvm](https://github.com/creationix/nvm) then use it to install node and npm.)
+
+  $ nvm install 0.10
```

Comment:
What issues were you having with other versions of node? If you were having problems with brew's default version, I'm all for locking down our node dependency. However, I'd prefer using a more recent version than 0.10. We're currently using v0.12.2 and haven't run into any problems.